### PR TITLE
Set the user timezone for tests

### PR DESCRIPTION
**Pull request by jdpgrailsdev**
## What
* Fix failing test on CI
* https://github.com/airbytehq/airbyte-internal-issues/issues/2157

## How
* Explicitly set the timezone for tests to `UTC` to avoid different timestamp values when testing locally vs CI

## Recommended reading order
1. `build.gradle`

### CODE REVIEW

File: airbyte-cdk/java/airbyte-cdk/db-sources/build.gradle [hunk 1]
```diff
@@ -16,9 +16,11 @@ plugins {
     id "java-library"
     id "java-test-fixtures"  // https://docs.gradle.org/current/userguide/java_testing.html#sec:java_test_fixtures
 }
-// TODO: Remove this after debugging
+
 test {
+    systemProperty 'user.timezone', 'UTC'
```
Comment (postamar) [inline airbyte-cdk/java/airbyte-cdk/db-sources/build.gradle:21]: This seems like it should be set repo-wide. Can you file an issue and assign me to it, please? I gotta run.

File: airbyte-cdk/java/airbyte-cdk/db-sources/build.gradle [hunk 2]
```diff
@@ -16,9 +16,11 @@ plugins {
...
     testLogging {
+        // TODO: Remove this after debugging
         showStandardStreams = true
```
Comment (postamar) [inline airbyte-cdk/java/airbyte-cdk/db-sources/build.gradle:24]: Should you be commenting this line out? I don't have an opinion either way, I don't know what it does, it just seems like an oversight.

**Review by postamar — APPROVED** [review]

**Comment by postamar** [comment]
FYI You need to remove the `@Disabled` annotation that I added as well

File: airbyte-cdk/java/airbyte-cdk/db-sources/build.gradle (hunk 1 above)
Comment (jdpgrailsdev) [reply airbyte-cdk/java/airbyte-cdk/db-sources/build.gradle:21]: @postamar Probably.  Will create a ticket.

Comment (jdpgrailsdev) [reply airbyte-cdk/java/airbyte-cdk/db-sources/build.gradle:21]: @postamar Done:  https://github.com/airbytehq/airbyte/issues/31060

File: airbyte-cdk/java/airbyte-cdk/db-sources/build.gradle (hunk 2 above)
Comment (postamar) [reply airbyte-cdk/java/airbyte-cdk/db-sources/build.gradle:24]: Oh, my bad, I didn't realize it had been uncommented a while ago. Please disregard my comment :)

File: airbyte-cdk/java/airbyte-cdk/db-sources/build.gradle (hunk 1 above)
Comment (postamar) [reply airbyte-cdk/java/airbyte-cdk/db-sources/build.gradle:21]: Thanks!

**Comment by airbyte-oss-build-runner** [comment]
##  source-mssql test report (commit [`8ce3d12412`](https://github.com/airbytehq/airbyte/pull/31057/commits/8ce3d1241275b4298fb0a172a261ad33261dd17f)) - ❌

⏲️  Total pipeline duration: 14.57s 

| Step   | Result   |
|--------|----------|

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_set-test-timezone/1696453295/8ce3d1241275b4298fb0a172a261ad33261dd17f/source-mssql/2.0.1/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:8ce3d1241275b4298fb0a172a261ad33261dd17f)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-mssql test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-mysql test report (commit [`8ce3d12412`](https://github.com/airbytehq/airbyte/pull/31057/commits/8ce3d1241275b4298fb0a172a261ad33261dd17f)) - ❌

⏲️  Total pipeline duration: 11mn53s 

| Step                                                         | Result   |
|:-------------------------------------------------------------|:---------|
| Build connector tar                                          | ✅        |
| Build source-mysql docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                    | ❌        |
| Java Connector Integration Tests                             | ✅        |
| Acceptance tests                                             | ✅        |
| Validate metadata for source-mysql                           | ✅        |
| Connector version semver check                               | ✅        |
| QA checks                                                    | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_set-test-timezone/1696453295/8ce3d1241275b4298fb0a172a261ad33261dd17f/source-mysql/3.1.1/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:8ce3d1241275b4298fb0a172a261ad33261dd17f)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-mysql test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-postgres-strict-encrypt test report (commit [`851dfc0e07`](https://github.com/airbytehq/airbyte/pull/31057/commits/851dfc0e073d361527efe12e978cfdee304c5565)) - ❌

⏲️  Total pipeline duration: 01mn24s 

| Step                                                 | Result   |
|:-----------------------------------------------------|:---------|
| Build connector tar                                  | ❌        |
| Validate metadata for source-postgres-strict-encrypt | ✅        |
| Connector version semver check                       | ✅        |
| QA checks                                            | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_set-test-timezone/1696454736/851dfc0e073d361527efe12e978cfdee304c5565/source-postgres-strict-encrypt/3.1.8/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:851dfc0e073d361527efe12e978cfdee304c5565)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-postgres-strict-encrypt test
```

**Comment by octavia-approvington** [comment]
Check that box <br/> TODO complete

**Review by octavia-approvington — APPROVED** [review]

**Comment by airbyte-oss-build-runner** [comment]
##  source-mongodb-v2 test report (commit [`851dfc0e07`](https://github.com/airbytehq/airbyte/pull/31057/commits/851dfc0e073d361527efe12e978cfdee304c5565)) - ❌

⏲️  Total pipeline duration: 13mn26s 

| Step                                                              | Result   |
|:------------------------------------------------------------------|:---------|
| Build connector tar                                               | ✅        |
| Build source-mongodb-v2 docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                         | ❌        |
| Java Connector Integration Tests                                  | ✅        |
| Acceptance tests                                                  | ✅        |
| Validate metadata for source-mongodb-v2                           | ✅        |
| Connector version semver check                                    | ✅        |
| QA checks                                                         | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_set-test-timezone/1696454736/851dfc0e073d361527efe12e978cfdee304c5565/source-mongodb-v2/1.0.1/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:851dfc0e073d361527efe12e978cfdee304c5565)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-mongodb-v2 test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-mysql test report (commit [`851dfc0e07`](https://github.com/airbytehq/airbyte/pull/31057/commits/851dfc0e073d361527efe12e978cfdee304c5565)) - ❌

⏲️  Total pipeline duration: 03mn38s 

| Step                                                         | Result   |
|:-------------------------------------------------------------|:---------|
| Build connector tar                                          | ✅        |
| Build source-mysql docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                    | ❌        |
| Java Connector Integration Tests                             | ✅        |
| Acceptance tests                                             | ✅        |
| Validate metadata for source-mysql                           | ✅        |
| Connector version semver check                               | ✅        |
| QA checks                                                    | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_set-test-timezone/1696454736/851dfc0e073d361527efe12e978cfdee304c5565/source-mysql/3.1.1/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:851dfc0e073d361527efe12e978cfdee304c5565)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-mysql test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-mysql-strict-encrypt test report (commit [`851dfc0e07`](https://github.com/airbytehq/airbyte/pull/31057/commits/851dfc0e073d361527efe12e978cfdee304c5565)) - ✅

⏲️  Total pipeline duration: 12mn13s 

| Step                                                                        | Result   |
|:----------------------------------------------------------------------------|:---------|
| Build connector tar                                                         | ✅        |
| Build source-mysql-strict-encrypt docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                                   | ✅        |
| Java Connector Integration Tests                                            | ✅        |
| Acceptance tests                                                            | ✅        |
| Validate metadata for source-mysql-strict-encrypt                           | ✅        |
| Connector version semver check                                              | ✅        |
| QA checks                                                                   | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_set-test-timezone/1696454736/851dfc0e073d361527efe12e978cfdee304c5565/source-mysql-strict-encrypt/3.1.1/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:851dfc0e073d361527efe12e978cfdee304c5565)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-mysql-strict-encrypt test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-postgres test report (commit [`851dfc0e07`](https://github.com/airbytehq/airbyte/pull/31057/commits/851dfc0e073d361527efe12e978cfdee304c5565)) - ❌

⏲️  Total pipeline duration: 56.05s 

| Step                                  | Result   |
|:--------------------------------------|:---------|
| Build connector tar                   | ❌        |
| Validate metadata for source-postgres | ✅        |
| Connector version semver check        | ✅        |
| QA checks                             | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_set-test-timezone/1696454736/851dfc0e073d361527efe12e978cfdee304c5565/source-postgres/3.1.9/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:851dfc0e073d361527efe12e978cfdee304c5565)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-postgres test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-mssql-strict-encrypt test report (commit [`851dfc0e07`](https://github.com/airbytehq/airbyte/pull/31057/commits/851dfc0e073d361527efe12e978cfdee304c5565)) - ❌

⏲️  Total pipeline duration: 307mn20s 

| Step                                                                        | Result   |
|:----------------------------------------------------------------------------|:---------|
| Build connector tar                                                         | ✅        |
| Build source-mssql-strict-encrypt docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                                   | ❌        |
| Java Connector Integration Tests                                            | ❌        |
| Acceptance tests                                                            | ❌        |
| Validate metadata for source-mssql-strict-encrypt                           | ✅        |
| Connector version semver check                                              | ✅        |
| QA checks                                                                   | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_set-test-timezone/1696454736/851dfc0e073d361527efe12e978cfdee304c5565/source-mssql-strict-encrypt/2.0.1/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:851dfc0e073d361527efe12e978cfdee304c5565)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-mssql-strict-encrypt test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-mssql test report (commit [`851dfc0e07`](https://github.com/airbytehq/airbyte/pull/31057/commits/851dfc0e073d361527efe12e978cfdee304c5565)) - ❌

⏲️  Total pipeline duration: 17.34s 

| Step   | Result   |
|--------|----------|

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_set-test-timezone/1696454736/851dfc0e073d361527efe12e978cfdee304c5565/source-mssql/2.0.1/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:851dfc0e073d361527efe12e978cfdee304c5565)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-mssql test
```

**Pull request merged**